### PR TITLE
docs: Added release notes for #5121

### PR DESCRIPTION
**Pull request by ogabrielides**
## What was done?
Added release notes for #5121.

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: doc/release-notes-5121.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Testnet Breaking Changes
+------------------------
+
+V20 will include two separates hard-forks.
```
Comment (thephez) [inline doc/release-notes-5121.md:4]:
```suggestion
V20 will include two separate hard-forks.
```

File: doc/release-notes-5121.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Testnet Breaking Changes
+------------------------
+
+V20 will include two separate hard-forks.
+- `v20` that activates Enhanced Hard Fork, LLMQ Randomness Beacon
+- `mn_rr` that activates Masternode Reward Location Reallocation.
```
Comment (thephez) [inline doc/release-notes-5121.md:6]:
```suggestion
- `mn_rr` that activates Masternode Reward Location Reallocation
```

File: doc/release-notes-5121.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Testnet Breaking Changes
+------------------------
+
+V20 will include two separate hard-forks.
+- `v20` that activates Enhanced Hard Fork, LLMQ Randomness Beacon
+- `mn_rr` that activates Masternode Reward Location Reallocation
+
+Note that `mn_rr` will be activated using the Enhanced Hard Fork mechanism.
```
[the remark this answers is not in the document]
Comment (ogabrielides) [reply doc/release-notes-5121.md:8]: @knst Keep this in mind

File: doc/release-notes-5121.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+The earliest activation date for `v20` is **TO BE DEFINED**.
+The earliest activation date for `mn_rr` is **TO BE DEFINED**.
```
[the remark this answers is not in the document]
Comment (ogabrielides) [reply doc/release-notes-5121.md]: @UdjinM6 Why?

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK for squash merge

**Review by thephez — APPROVED** [review]

**Pull request merged**